backend/models/testSession.model.ts

```diff
@@ -16,7 +16,7 @@ export interface Result {
    * - [numeric answer] for short answer
    * - [index] for multiple choice
    * - list of indices for multiple select
-   * - [numerator, denominator] for fraction
+   * - [numerator, denominator] or [whole number, numerator, denominator] for fraction
```

Comment:
kept them as variable lengths rather than [whole number / null, numerator, denominator] to avoid changing types to include `null`